p-process.yml

```diff
@@ -187,6 +187,14 @@ components:
           By default it is set to system locale default (where the service is running)."
           type: string
           example: "MM/dd/yyyy"
+        measureSystem:
```

Comment:
add enum values as in https://github.com/regulaforensics/DocumentReader-web-openapi/blob/master/e-document-type.yml